### ISSUE TITLE
Make issues

### ISSUE THREAD

**Issue by IdrisMiles** [Gmadges/walkr-client#1]
## Problem

Currently we don't have a way to assign work to each other or track who is working on what.

## Possible Solution

Might be nice to make issues for the work that needs to be done and assign them to each other, so we can track progress and work efficiently?

Would love to hear thoughts on this?

**Comment by Gmadges** [comment]
@IdrisMiles  Yep I agree. that sounds good. I can start doing that with this repo. Just to let you know. anything server related will be in this [repo.](https://github.com/Gmadges/walkr-server). I think its best to keep the front end and back end separate.
